src/v2/guide/filters.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Filters
+type: guide
+order: 305
+---
+
+Vue.js allows you to define filters that can be used to apply common text formatting. Filters are usable in two places: **mustache interpolations and `v-bind` expressions**. Filters should be appended to the end of the JavaScript expression, denoted by the "pipe" symbol:
+
+``` html
+<!-- in mustaches -->
+{{ message | capitalize }}
+
+<!-- in v-bind -->
+<div v-bind:id="rawId | formatId"></div>
+```
+
+<p class="tip">Vue 2.x filters can only be used inside mustache interpolations and `v-bind` expressions (the latter supported in 2.1.0+), because filters are primarily designed for text transformation purposes. For more complex data transforms in other directives, you should use [Computed properties](computed.html) instead.</p>
```

Comment:
This paragraph repeats the way of using filters.
The part "(the latter supported in 2.1.0+)" can be moved to the intro paragraph, so this can be: "Filters are primarily designed for text transformation purposes. For more complex data transforms....."